apps/site-landing/src/components/RangeFieldAndScale/Slider.stories.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Story } from '@storybook/react';
+import type { NumberFormatOptionsStyle } from '@formatjs/ecma402-abstract';
+import { RangeFieldAndScale } from '.';
+
+const meta = {
+  title: 'Landing/RangeFieldAndScale',
+  component: RangeFieldAndScale,
+  parameters: {
+    backgrounds: {
+      default: 'graphColor',
+      values: [
+        { name: 'graphColor', value: '#138175' },
+      ],
+    },
+  },
+  argTypes: { onChange: { action: 'onChange' } },
+  args: {
+    label: "Label",
+    currency: "CAD" as "CAD",
```

Comment:
That's a surprising line. What is it doing?

Reply:
ha!  Yeah, its kinda WTF :-).

Our slider displays a currency. One day we might have multiple currencies, so it accepts an argument (which kind of currency).  But we want to type-check the type of currency, so the only one accepted is type "CAD".  But when we put the argument here of "CAD", TS thinks it's of type string unless I explicitly override it with type "CAD".

If I put "CAD" directly as argument to component, I wouldn't need it, it's just cause I let it come in as an argument.

All technically correcter, but prolly more work than it's worth.